src/Spout/Writer/Style/StyleBuilder.php

```diff
@@ -109,15 +109,15 @@ public function setFontName($fontName)
     }
 
     /**
-     * Makes the text wrap in the cell if it's too long or
-     * on multiple lines.
+     * Makes the text wrap in the cell if requested
      *
      * @api
+     * @param boolean $shouldWrap Should the text be wrapped
```

Comment:
`@param bool` (for consistency, as we've been using "bool" instead of "boolean" everywhere and this is what PHP uses too)